extensions/data-workspace/package.vscode.json

```diff
@@ -1,4 +1,5 @@
 {
 	"name": "data-workspace-vscode",
-	"publisher": "ms-mssql"
+	"publisher": "ms-mssql",
+	"license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/extensions/data-workspace/LICENSE.txt"
```

Comment:
Per https://code.visualstudio.com/api/references/extension-manifest#fields this should be `SEE LICENSE IN LICENSE.TXT`